src/pages/api/profile/get-profile.ts

```diff
@@ -53,7 +53,10 @@ const handler: Handler = async (request, response) => {
     const { data: profileUserInfo } = await profileClient
       .getProfile()
```

Comment:
I think we should have two separate endpoints for the Connect.Profile data:
- `api/connect-profile/get-profile.ts`
- `api/connect-profile/get-addresses.ts`
And make two calls in the pages GSSP using those so we can more easily deal with errors and have a better separation of concerns.